recipes/rgccacmd/meta.yaml

```diff
@@ -1,23 +1,24 @@
 {% set name = "rgcca" %}
-{% set version = "3.0.0" %}
+{% set version = "3.0.1" %}
 
 package:
   name: {{ name }}cmd
   version: {{ version }}
 
 source:
   url: https://github.com/rgcca-factory/{{ name|upper }}/archive/{{ version }}.tar.gz
-  sha256: 2ee056b54a4564fdff73e02967d18132a34443ca1caed4675ae620237e43104a
+  sha256: cd18078603b8eae54b649ae45cb33143cd9049af33d749a3e3ebd795fcf890db
 
 build:
   noarch: generic
-  number: 1
+  number: 0
   rpaths:
     - lib/
     - lib/R/lib/
 
 requirements:
   host:
+    - wget
```

Comment:
you need that in the run section, do you?

Reply:
It seems to be the case with my new version. Without it, the last commit failed.

Reply:
This was to download the command line interface that was migrated to another directory in the latest version. Once installed during the build, it should be fine, but I can add it in the run section to be sure. Strangely the similar version of conda in my personal directory doesn't need "wget" (https://anaconda.org/icm-iconics/r-rgcca).